app/views/consents/index.html.erb

```diff
@@ -8,24 +8,21 @@
 
 <%= h1 "Check consent responses", class: "govuk-heading-l" %>
 
-<%= govuk_tabs title: "Vaccinations", classes: 'nhsuk-tabs' do |c|
-  c.with_tab(label: "Consent given (#{ @tabs[:consent_given?].size })",
-             classes: 'nhsuk-tabs__panel') do
-    @tabs[:consent_given?].size.to_s
-  end
-
-  c.with_tab(label: "Consent refused (#{ @tabs[:consent_refused?].size })",
-             classes: 'nhsuk-tabs__panel') do
-    @tabs[:consent_refused?].size.to_s
-  end
-
-  c.with_tab(label: "Consent conflicts (#{ @tabs[:consent_conflicts?].size })",
-             classes: 'nhsuk-tabs__panel') do
-    @tabs[:consent_conflicts?].size.to_s
+<% def consent_tab(c, label, data)
+  c.with_tab(label: "#{label} (#{data.size })", classes: 'nhsuk-tabs__panel') do
```

Comment:
What is `c` here? Would it ruin your golf game if we gave it a more descriptive name?

Reply:
```suggestion
  c.with_tab(label: "#{label} (#{data.size})", classes: 'nhsuk-tabs__panel') do
```

Reply:
`c` stands for `component` I think https://govuk-components.netlify.app/components/tabs/